Active Documents/contenteditable-disabled.html

```diff
@@ -0,0 +1,192 @@
+                <li><code>formatfontsize</code></li>
+                <li><code>formatfontcolor</code></li>
+                <li><code>formatindent</code></li>
+                <li><code>inserthorizontalrule</code></li>
+                <li><code>insertorderedlist</code></li>
+                <li><code>insertunorderedlist</code></li>
+                <li><code>formatitalic</code></li>
+                <li><code>formatoutdent</code></li>
+                <li><code>formatremove</code></li>
+                <li><code>formatstrikethrough</code></li>
+                <li><code>formatsubscript</code></li>
+                <li><code>formatsuperscript</code></li>
+                <li><code>formatunderline</code></li>
+            </ul>
+        </p>
+        <p class="issue" title="missing fontSize and insertImage">
+            fontSize / insertImage is missing from beforeInput.  Why?  It seems that it should be listed to create a more complete set.
+            <a href="https://github.com/w3c/editing/issues/249">Issue 249</a>
+        </p>
+        <h3>
+            <code>contenteditable-disabled</code> Allowed Token Shortcuts.
+        </h3>
+        <p>
+            The following tokens are also supported by the <code>contenteditable-disabled</code> attribute and serve as shorthands for specifying a set of tokens from contenteditable-disabled Allowed Tokens:
+        </p>
+        <p>
+            <ul>
+                <li><code>formatjustify</code>: combines <code>formatjustifycenter</code>, <code>formatjustifyfull</code>, <code>formatjustifyleft</code>, and <code>formatjustifyright</code>.</li>
+                <li><code>formatfont</code>: combines <code>formatfontname</code>, <code>formatfontsize</code>, and <code>formatfontcolor</code></li>
+                <li><code>formatdent</code>: combines <code>formatindent</code> and <code>formatoutdent</code></li>
+                <li><code>insertlist</code>: combines <code>insertorderedlist</code> and <code>insertunorderedlist</code></li>
+                <li><code>formatscript</code>: combines <code>formatsubscript</code> and <code>formatsuperscript</code></li>
+            </ul>
+        </p>
+        <p class="note">
+            Note that this specification does not have a shortcut to disable all editing UI.  Authors should consider using a <code>textarea</code> or <code>contenteditable="plaintext-only"</code> for this purpose.
+        </p>
+    </section>
+    <section id="element-contenteditable-mixin">
+        <h2>Extensions to the <dfn>ElementContentEditable</dfn> mixin</h2>
+        <pre class="idl">
+            [Exposed=Window]
+            interface ElementContentEditable {
+                [SameObject, PutForwards=value] readonly attribute DOMTokenList contentEditableDisabled;
+            };
+        </pre>
+        <p>
+            The <code>contentEditableDisabled</code> WebIDL property reflects the contenteditable-disabled DOM content attribute.
```

Comment:
There are some things in this spec that should be definitions and others that should be linking to those definitions.  Need to determine how to make that happen in ReSpec.  For example, contenteditable-disabled DOM content attribute should probably be linking back to the definition for that phrase.

Reply:
yeah, I've created an issue to fix DOMTokenList linking, I can use it for all dfns related fixes.

Reply:
> Nice work @gked and @BoCupp-Microsoft ! I noticed you removed `supported`. This was something we had added after discussing whether to blacklist or to whitelist to kind of allow whitelisting anyway while giving a preference for blacklisting.
> 
> The reason this is important to JS editors is that it took several of the JS editor companies several months to discover that the Mac touch bar had been launched. During this time, their users would use the buttons of this bar and nothing would happen in the users editors or even worse - the editors were breaking. The same can happen again with new devices launches with new versions of browsers and operating systems that not every development team is aware of immediately. For that reason there was a demand for a way to query all the supported options that then can be disabled subsequently.
> 
> Maybe we need to open a ticket about this?

Sounds good, I created an [issue ](https://github.com/w3c/editing/issues/252)and tagged with Agenda+ to be discussed tomorrow.

Reply:
When `xref` option is enabled, DOMTokenList will link correctly :)